packages/mobile/src/screens/search-screen-v2/SearchCategoriesAndFilters.tsx

```diff
@@ -93,10 +94,25 @@ export const SearchCategoriesAndFilters = () => {
             <FilterButton
               key={filter}
               size='small'
-              // value={value}
-              label={filterInfoMap[filter].label}
+              value={
+                filters[filter] !== undefined
+                  ? String(filters[filter])
+                  : undefined
+              }
+              label={
+                typeof filters[filter] === 'string'
+                  ? String(filters[filter])
+                  : filterInfoMap[filter].label
```

Comment:
It's nice that we don't have to show the emoji for mood like on web

Reply:
I think that we do actually. I just updated this to have some feedback that the filters were working, but this should be updated later to have more presentable labels when the value is set